packages/admin/admin/src/translator/config/useContentTranslationServiceProvider.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+import { ContentTranslationServiceContext, TranslationConfig } from "./ContentTranslationServiceContext";
+
+export function useContentTranslationServiceProvider(): TranslationConfig {
+    const context = React.useContext(ContentTranslationServiceContext);
+    return context ?? {};
```

Comment:
context is never undefined because you provide a default value. And if it would be, {} would be incorrect.